code/datums/diseases/advance/symptoms/tumors.dm

```diff
@@ -82,6 +75,9 @@
 		if(missing_limbs.len > 0)
 			var/limb_to_regenerate = pick(missing_limbs)
 			M.regenerate_limb(limb_to_regenerate,TRUE);
+			var/obj/item/bodypart/new_limb = M.get_bodypart(limb_to_regenerate);
```

Comment:
is this the right way to make a really fragile limb? or does brute + burn have to add up to < 50 as max_damage is 50?